libraries/AP_NavEKF2/AP_NavEKF2_VehicleStatus.cpp

```diff
@@ -469,7 +469,8 @@ void NavEKF2_core::setTouchdownExpected(bool val)
 
 // Set to true if the terrain underneath is stable enough to be used as a height reference
 // in combination with a range finder. Set to false if the terrain underneath the vehicle
-// cannot be used as a height reference
+// cannot be used as a height reference. Use to prevent range finder operation otherwise
+// enabled by the combination of EK2_RNG_AID_HGT and EK2_RNG_USE_SPD parameters.
 void NavEKF2_core::setTerrainHgtStable(bool val)
 {
     terrainHgtStableSet_ms = imuSampleTime_ms;
```

Comment:
It's an existing issue but it looks like terrainHgtStableSet_ms is never used so we might as well remove it.